packages/amplify-schema-validator/src/validators/field-is-defined-once.ts

```diff
@@ -0,0 +1,40 @@
+import {
+  DocumentNode,
+  Kind,
+  ObjectTypeDefinitionNode,
+} from 'graphql';
+import { InvalidDirectiveError } from '../exceptions/invalid-directive-error';
+
+/**
+   * Field is defined once in a model
+   *
+   * @param schema graphql schema
+   * @returns true
```

Comment:
Returns true if... - would you mind finishing this doc string up?

Reply:
Fixed